.storybook/config.js

```diff
@@ -0,0 +1,8 @@
+import { addDecorator } from "@storybook/react";
+
+addDecorator((s) => (
```

Comment:
Just a suggestion, but abbreviation to a single letter like `s` as variable can be hard to understand if you're not familiar with a library